.github/workflows/ci-workflow.yml

```diff
@@ -19,12 +22,6 @@ jobs:
           cache-dependency-path: 'pnpm-lock.yaml'
           node-version-file: '.nvmrc'
       - run: pnpm install --frozen-lockfile
-
-      - name: Build core packages
-        run: pnpm build --filter=@suspensive/react --filter=@suspensive/react-query
-
-      - name: Check eslint
-        run: pnpm lint
-
-      - name: Test by jest
-        run: pnpm test
+      - run: pnpm build --filter=@suspensive/react --filter=@suspensive/react-query
```

Comment:
I think the process of building is also a kind of `ci`,
Is there a reason why you separated the 13th line without including the build in the array?

Reply:
Because `pnpm test:tsd`, `pnpm lint` script have dependency on *.d.ts files built by `pnpm build --filter=@suspensive/react --filter=@suspensive/react-query`

You can check why I separated it if you run lint, test:tsd after deleting dist, esm directory built by `pnpm build` in your local machine

Reply:
I thank you for explaning well
I understood 100% ^_^